partials/language-specific-deploy/php.md

```diff
@@ -432,10 +426,9 @@ To enable this feature, you need to:
  - create an [environment variable](#setting-up-environment-variables-on-clever-cloud) named `SESSION_TYPE` with the value `redis`.
 
 {{< alert "warning" "Warning:" >}}
-    <p>You must have a [Redis]({{< ref "deploy/addon/redis.md" >}}) add-on
-    [linked with your application](#linking-a-database-or-any-other-add-on-to-your-application)
-    to enable PHP session storage in Redis.<br />
-    If no Redis add-on is linked with your application, the deployment will fail.</p>
+    You must have a [Redis]({{< ref "deploy/addon/redis.md" >}}) add-on [linked with your application](#linking-a-database-or-any-other-add-on-to-your-application) to enable PHP session storage in Redis.
```

Comment:
Same here for all the markdown stuff

Reply:
There is also another warning that isn't in this diff that starts with "Environment variables are displayed" that contains markdown tags inside the HTML, could you also fix it?

Reply:
Same thing here and I don't get this warning, what command do you run ?